### PR TITLE
BROKERS: LoggingBroker - LogInformation 

### LINKED ISSUE #27: BROKERS: LoggingBroker - LogInformation

### PR DESCRIPTION
**Pull request by Lezune**
Closes #27.

### CODE REVIEW

File: GitFyle.Core.Api/Brokers/Loggings/ILoggingBroker.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+using System;
```
Comment (glhays) [inline GitFyle.Core.Api/Brokers/Loggings/ILoggingBroker.cs:4]: Add a space between line 3 and 4 please.

File: GitFyle.Core.Api/Program.cs [hunk 2]
```diff
@@ -31,6 +32,9 @@ public static void Main(string[] args)
                 ILoggingBroker,
                 LoggingBroker>();
 
+                IDateTimeBroker,
```
Comment (glhays) [inline GitFyle.Core.Api/Program.cs:35]: Register the DateTimeBroker with the AddTransient extenstion method, will be the same as the registration done with the Storage and Logging Brokers.

Comment (Lezune) [reply GitFyle.Core.Api/Program.cs:36]: This has been updated. Please review to confirm issue has been resolved.

Comment (glhays) [reply GitFyle.Core.Api/Program.cs:36]: Good catch on noticing this wasn't handled.

File: GitFyle.Core.Api/Brokers/Loggings/LoggingBroker.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+using Microsoft.Extensions.Logging;
+
+namespace GitFyle.Core.Api.Brokers.Loggings
+{
+    public class LoggingBroker : ILoggingBroker
+    {
+        private readonly ILogger logger;
+
+        public LoggingBroker(ILogger logger) => this.logger = logger;
```
Comment (cjdutoit) [inline GitFyle.Core.Api/Brokers/Loggings/LoggingBroker.cs:13]:
```suggestion
        public LoggingBroker(ILogger logger) => 
            this.logger = logger;   
```

Comment (Lezune) [reply GitFyle.Core.Api/Brokers/Loggings/LoggingBroker.cs]: This has been updated. Please review to confirm issue has been resolved.

**Review by cjdutoit — APPROVED** [review]

**Pull request merged**